diff --git a/views_data_export.info b/views_data_export.info
index 96e4e13..fde9aa1 100644
--- a/views_data_export.info
+++ b/views_data_export.info
@@ -12,6 +12,7 @@ files[] = plugins/views_data_export_plugin_display_export.inc
 files[] = plugins/views_data_export_plugin_style_export.inc
 files[] = plugins/views_data_export_plugin_style_export_csv.inc
 files[] = plugins/views_data_export_plugin_style_export_xml.inc
+files[] = plugins/views_data_export_handler_field_raw_xml.inc
 
 ; Tests
 files[] = "tests/base.test"
diff --git a/views_data_export.module b/views_data_export.module
index 7bea5df..5d09a03 100644
--- a/views_data_export.module
+++ b/views_data_export.module
@@ -171,7 +171,6 @@ function _views_data_export_batch_process($export_id, $display_id, &$context) {
 }
 
 
-
 /**********/
 /** CRUD **/
 /**********/
@@ -273,3 +272,4 @@ function views_data_export_view_clear($export_id) {
     ->execute();
   }
 }
+
diff --git a/views_data_export.views.inc b/views_data_export.views.inc
index c433c79..7da258e 100644
--- a/views_data_export.views.inc
+++ b/views_data_export.views.inc
@@ -5,6 +5,26 @@
  */
 
 /**
+ * Implements hook_views_data().
+ */
+function views_data_export_views_data() {
+  $data['views_data_export_global']['table']['group'] = t('Global');
+  $data['views_data_export_global']['table']['join'] = array(
+    '#global' => array(),
+  );
+  
+  $data['views_data_export_global']['raw_xml'] = array(
+    'title' => t('Raw XML'),
+    'help' => t('Ability to add XML formatted text into a field. Useful for using formatted XML from other fields and displaying them here using tokens. <em>Note: No character escaping is applied to the value of this field. Use with caution.</em>.'),
+    'field' => array(
+      'handler' => 'views_data_export_handler_field_raw_xml',
+    ),
+  );
+
+  return $data;
+}
+
+/**
  * Implementation of hook_views_plugins().
  */
 function views_data_export_views_plugins() {
